web/src/main/ui/app/components/flows-new/edit-flow/ingest/ingest.component.ts

```diff
@@ -29,7 +29,8 @@ export class IngestComponent implements OnInit {
   @Output() saveStep = new EventEmitter();
 
   ngOnInit(): void {
-    this.checkDefaults();
+    // Not needed
+    // this.checkDefaults();
```

Comment:
To avoid conflicts I suggest rollback ingest options. My PR will take care of that.